<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleEnergyPitchIJ</class>
 <widget class="QMainWindow" name="SingleEnergyPitchIJ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Single-energy, pitch, image</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="sliderEnergy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblREEnergy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Runaway energy</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnergy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnergyMin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEnergyMax">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>âˆž</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetDistPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>531</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sliderPitchAngle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblREPitchAngle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch angle parameter</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPitchAngleMin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPitchAngleMax">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>200</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbFilename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>431</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Green's function</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowse">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPitchAngle">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>70</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Super particle</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDomPitch">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblOverlay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlay</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbOverlay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>431</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseOverlay">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>610</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderOverlay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>644</y>
      <width>511</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblOverlayMin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOverlayMax">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>670</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblOverlay25">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>670</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>25%</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOverlay50">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>670</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOverlay75">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>670</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>75%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveSuper">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save super-particle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveBoth">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save both</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbUnderlay">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>580</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reversed</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
